python_resources/vscode_python_test_adapter/pytest/discovery_output_plugin.py

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Extracting this script to its own `.py` file also had the nice benefit of syntax highlighting and all the other benefits that come with being recognized by the IDE as a Python file/code.